### PR TITLE
Add common crates

### PR DESCRIPTION
**Pull request by vovac12**
Some crates from mainnet

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target/
```
Comment (wer1st) [inline .gitignore]: add empty line

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "pallets/trustless-eth-bridge/types/tests/fixtures/tests"]
+	path = pallets/trustless-eth-bridge/types/tests/fixtures/tests
+	url = https://github.com/ethereum/tests.git
```
Comment (wer1st) [inline .gitmodules]: add empty line

**Review by wer1st — APPROVED** [review]

File: pallets/trustless-eth-bridge/eth-app/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+            recipient: H160,
+            amount: BalanceOf<T>,
+        ) -> DispatchResult {
+            let who = ensure_signed(origin)?;
+            Pallet::<T>::burn_inner(who, network_id, recipient, amount)?;
+            Ok(())
+        }
+
+        #[pallet::weight(<T as Config>::WeightInfo::mint())]
+
```
Comment (N1ghtStorm) [inline pallets/trustless-eth-bridge/eth-app/src/lib.rs]: is this empty line necessary?(same for whole file)

File: pallets/trustless-eth-bridge/migration-app/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+        TokenRegisteredWithAnotherAddress,
+        /// Call encoding failed.
+        CallEncodeFailed,
+    }
+
+    #[pallet::call]
+    impl<T: Config> Pallet<T> {
+        // Send ERC20 tokens to ERC20 App address and register tokens.
+        #[pallet::weight(<T as Config>::WeightInfo::burn())]
+
```
Comment (N1ghtStorm) [inline pallets/trustless-eth-bridge/migration-app/src/lib.rs]: Empty line

**Review by N1ghtStorm — CHANGES_REQUESTED** [review]

**Review by f33r0 — APPROVED** [review]

**Pull request merged**